modules/getting-started/proc-customize-rhdh-homepage.adoc

```diff
@@ -1,12 +1,394 @@
+----
+
+.Available props
+
+|===
+| Prop | Default | Description
+
+| `path`
+| `/search`
+| Override the linked search path if needed
+
+| `queryParam`
+| `query`
+| Override the search query parameter name if needed
+|===
+--
+
+Quick access::
++
+--
+[source,yaml]
+----
+dynamicPlugins:
+  frontend:
+    janus-idp.backstage-plugin-dynamic-home-page:
+      mountPoints:
+        - mountPoint: home.page/cards
+          importName: QuickAccessCard
+          config:
+            layouts:
+              xl: { h: 8 }
+              lg: { h: 8 }
+              md: { h: 8 }
+              sm: { h: 8 }
+              xs: { h: 8 }
+              xxs: { h: 8 }
+----
+
+.Available props
+
+|===
+| Prop | Default | Description
+
+| `title`
+| `Quick Access`
+| Override the linked search path if needed
+
+| `path`
+| none
+| Override the search query parameter name if needed
+|===
+--
+
+Headline::
++
+--
+[source,yaml]
+----
+dynamicPlugins:
+  frontend:
+    janus-idp.backstage-plugin-dynamic-home-page:
+      mountPoints:
+        - mountPoint: home.page/cards
+          importName: Headline
+          config:
+            layouts:
+              xl: { h: 1 }
+              lg: { h: 1 }
+              md: { h: 1 }
+              sm: { h: 1 }
+              xs: { h: 1 }
+              xxs: { h: 1 }
+            props:
+              title: Important info
+----
+
+.Available props
+
+|===
+| Prop | Default | Description
+
+| `title`
+| none
+| Title
+|===
+--
+
+Markdown::
++
+--
+[source,yaml]
+----
+dynamicPlugins:
+  frontend:
+    janus-idp.backstage-plugin-dynamic-home-page:
+      mountPoints:
+        - mountPoint: home.page/cards
+          importName: SearchBar
+          config:
+            layouts:
+              xl: { w: 10, h: 1, x: 1 }
+              lg: { w: 10, h: 1, x: 1 }
+              md: { w: 10, h: 1, x: 1 }
+              sm: { w: 10, h: 1, x: 1 }
+              xs: { w: 12, h: 1 }
+              xxs: { w: 12, h: 1 }
+----
+--
+
+Placeholder::
++
+--
+[source,yaml]
+----
+dynamicPlugins:
+  frontend:
+    janus-idp.backstage-plugin-dynamic-home-page:
+      mountPoints:
+        - mountPoint: home.page/cards
+          importName: SearchBar
+          config:
+            layouts:
+              xl: { w: 10, h: 1, x: 1 }
+              lg: { w: 10, h: 1, x: 1 }
+              md: { w: 10, h: 1, x: 1 }
+              sm: { w: 10, h: 1, x: 1 }
+              xs: { w: 12, h: 1 }
+              xxs: { w: 12, h: 1 }
+----
+--
+
+Catalog starred entities::
++
+--
+[source,yaml]
+----
+dynamicPlugins:
+  frontend:
+    janus-idp.backstage-plugin-dynamic-home-page:
+      mountPoints:
+        - mountPoint: home.page/cards
+          importName: SearchBar
+          config:
+            layouts:
+              xl: { w: 10, h: 1, x: 1 }
+              lg: { w: 10, h: 1, x: 1 }
+              md: { w: 10, h: 1, x: 1 }
+              sm: { w: 10, h: 1, x: 1 }
+              xs: { w: 12, h: 1 }
+              xxs: { w: 12, h: 1 }
```

Comment:
This is also the searchbar example, please replace it with:

```yaml
dynamicPlugins:
  frontend:
    janus-idp.backstage-plugin-dynamic-home-page:
      mountPoints:
        - mountPoint: home.page/cards
          importName: CatalogStarredEntitiesCard
```

Reply:
Hi @jerolimov, for this yaml and the one for the featured docs, does it mean that we are leaving out the layouts?